### ISSUE TITLE
New rule: Element marked as decorative is not exposed

### ISSUE THREAD

**Issue by carlosapaduarte** [w3c/wcag-act#493]
The ACT-Rules community group has completed work on "Element marked as decorative is not exposed". We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: Element marked as decorative is not exposed
**Rule id**: 46ca7f
**Link to the rule**: https://act-rules.github.io/rules/46ca7f
**License**: W3C Software And Document Notice And License
**Implementors**:
- Alfa
- QualWeb

If you'd like to contact me, please e-mail me at caduarte@campus.ul.pt. (optional: You can also reach me on the W3C Community Slack, my name there is Carlos Duarte)

**Comment by maryjom** [comment]
[Survey](https://www.w3.org/2002/09/wbs/93339/ACTDecorativeElement/) is due on 3 December due to Thanksgiving holiday in the U.S.

**Comment by maryjom** [comment]
See [Issue 504](https://github.com/w3c/wcag-act/issues/504) due to comments from survey.
